Tests/LeafKitTests/LeafTests.swift

```diff
@@ -195,69 +196,54 @@ final class LeafTests: XCTestCase {
         try XCTAssertEqual(render(template, ["namelist": [.string("Tanner")]]), expectedName)
         try XCTAssertEqual(render(template), expectedNoName)
     }
-//
-//    func testEscapeExtraneousBody() throws {
-//        let template = """
-//        extension #("User") \\{
-//
-//        }
-//        """
-//        let expected = """
-//        extension User {
-//
-//        }
-//        """
-//        try XCTAssertEqual(renderer.testRender(template, .null), expected)
-//    }
-//
-//
-//    func testEscapeTag() throws {
-//        let template = """
-//        #("foo") \\#("bar")
-//        """
-//        let expected = """
-//        foo #("bar")
-//        """
-//        try XCTAssertEqual(renderer.testRender(template, .null), expected)
-//    }
-//
-//    func testCount() throws {
-//        let template = """
-//        count: #count(array)
-//        """
-//        let expected = """
-//        count: 4
-//        """
-//        let context = TemplateData.dictionary(["array": .array([.null, .null, .null, .null])])
-//        try XCTAssertEqual(renderer.testRender(template, context), expected)
-//    }
-//
-//    func testNestedSet() throws {
-//        let template = """
-//        #if(a){#set("title"){A}}title: #get(title)
-//        """
-//        let expected = """
-//        title: A
-//        """
-//
-//        let context = TemplateData.dictionary(["a": .bool(true)])
-//        try XCTAssertEqual(renderer.testRender(template, context), expected)
-//    }
-//
-//    func testDateFormat() throws {
-//        let template = """
-//        Date: #date(foo, "yyyy-MM-dd")
-//        """
-//
-//        let expected = """
-//        Date: 1970-01-16
-//        """
-//
-//        let context = TemplateData.dictionary(["foo": .double(1_337_000)])
-//        try XCTAssertEqual(renderer.testRender(template, context), expected)
-//
-//    }
-//
+
+//    func testEscapeExtraneousBody()... removed as obviated by syntax change
+
+    func testEscapeTag() throws {
+        let template = """
+        #("foo") \\#("bar")
+        """
+        let expected = """
+        foo #("bar")
+        """
+        try XCTAssertEqual(render(template, [:]), expected)
+    }
+    
+    // TODO: Reimplement #count
+    func _testCount() throws {
+        let template = """
+        count: #count(array)
+        """
+        let expected = """
+        count: 4
+        """
+        try XCTAssertEqual(render(template, ["array": ["","","",""]]), expected)
+    }
+    
+    // TODO: Are set/get totally deprecated?
```

Comment:
We should probably continue to support these. In Leaf 3 `#get` was a way to get unescaped strings. For example, you could store HTML content in the context and pull it out using `#get`. `#set` was used mostly for template inheritance which we don't need anymore. It could add data to the context. This might be useful to have still if it's easy enough to implement.

Reply:
Happy to put them back in as conveniences, just want to clarify intended behavior - like I'd love to just have
`#(x = "Hello World")` and `#(x)` for single tags and have set be body-setting only `#set(x):Hello World#endset` - set/get as single tags being kept for backwards-convenience only. And in both cases it should be globally-variable scoping, correct? As opposed to import/export which scope to the extended template.

Reply:
Clarification though? I'm not clear on why we'd worry about escaping during these anyway in Leaf4 - was Leaf3 escaping all stringLiterals?

Reply:
Addressing/adding in future PR concentrating on Tags